package.json

```diff
@@ -85,8 +87,15 @@
     "@adobe/helix-status": "7.1.2",
     "@adobe/openwhisk-action-logger": "2.2.0",
     "@adobe/openwhisk-action-utils": "4.2.2",
+    "cz-conventional-changelog": "^3.1.0",
     "fs-extra": "9.0.0",
     "glob-to-regexp": "0.4.1",
+    "openwhisk": "^3.21.1",
```

Comment:
why is this a dev dependency?

Reply:
Just a guess: because it's pre-installed on the container but needed during testing. Making it a regular dependency would be cleaner, though.

Reply:
I think this is outdated now

Reply:
it's about `openwhisk`